app/src/main/java/com/x8bit/bitwarden/data/vault/datasource/network/model/SecureNoteTypeJson.kt

```diff
@@ -21,4 +21,6 @@ enum class SecureNoteTypeJson {
 private class SecureNoteTypeSerializer : BaseEnumeratedIntSerializer<SecureNoteTypeJson>(
     className = "SecureNoteTypeJson",
     values = SecureNoteTypeJson.entries.toTypedArray(),
+    // Because GENERIC is the only valid value we default to it if an unknown value is received.
+    default = SecureNoteTypeJson.GENERIC,
```

Comment:
@vvolkgang @fedemkr Thoughts on defaulting this to `GENERIC`? It's a little heavy handed, but `GENERIC` is the only known valid note type. This not only fixes deserialization in the client, but it also corrects the record if the user updates the item.

Reply:
In the current state of all clients/server I think it's safe to do so. However, we need to keep this in mind if the `SecureNoteType` were to add any more values in the future which could break some logic so I'd ask internally if there are any plans on adding more types in the near/mid term future; if not I'd say go for it 😄